### PR TITLE
Msort

### PR DESCRIPTION
**Pull request by prachigupta28**
HI @sanjay1701

### CODE REVIEW

File: src/com/company/SampleClass.java [hunk 1]
```diff
@@ -15,18 +15,18 @@ public static void mergeSort(int[] a, int n) {
         if (n < 2) {
             return;
         }
-        int mid = n / 2;
```
Comment (sanjay1701) [inline src/com/company/SampleClass.java:18]: mid seems short and simple name

File: src/com/company/SampleClass.java [hunk 2]
```diff
@@ -48,9 +48,9 @@ public static void merge(int[] a, int[] l, int[] r, int left, int right) {
 
     @Test
     public void shouldMerge() {
-        int[] actual = {5, 1, 6, 2, 3, 4};
-        int[] expected = {1, 2, 3, 4, 5, 6};
-        SampleClass.mergeSort(actual, actual.length);
-        assertArrayEquals(expected, actual);
+        int[] actualArray = {5, 1, 6, 2, 3, 4,7};
```
Comment (sanjay1701) [inline src/com/company/SampleClass.java:51]: Please  format the code

**Review by sanjay1701 — CHANGES_REQUESTED** [review]

File: src/com/company/SampleClass.java (hunk 2 above)
Comment (prachigupta28) [reply src/com/company/SampleClass.java:51]: sure

File: src/com/company/SampleClass.java (hunk 1 above)
Comment (prachigupta28) [reply src/com/company/SampleClass.java:18]: implemented review comment

**Review by sanjay1701 — APPROVED** [review]

**Pull request merged**